src/ripple/app/main/Application.cpp

```diff
@@ -1085,16 +1085,16 @@ bool ApplicationImp::setup()
     }
 
     {
-        PublicKey validationPub;
-        SecretKey validationPriv;
+        PublicKey valPublic;
+        SecretKey valSecret;
         std::string manifest;
         if (config().exists (SECTION_VALIDATOR_TOKEN))
         {
             if (auto const token = ValidatorToken::make_ValidatorToken (
                 config().section (SECTION_VALIDATOR_TOKEN).lines ()))
             {
-                validationPriv = token->validationPrivate;
-                validationPub = derivePublicKey (KeyType::secp256k1, validationPriv);
+                valSecret = token->validationPrivate;
```

Comment:
Do you think the `"validation_private_key"` json field in the token should also be renamed?
https://github.com/wilsonianb/rippled/blob/rename-manifests/src/ripple/app/misc/impl/Manifest.cpp#L176
One of the reasons I had been using "private" is that the key uses the `TOKEN_NODE_PRIVATE` base58 encoding.
https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/tokens.h#L33